pkg/metrics/subsystems.go

```diff
@@ -8,6 +8,7 @@ const (
 	CentralSubsystem    Subsystem = "central"
 	SensorSubsystem     Subsystem = "sensor"
 	ComplianceSubsystem Subsystem = "compliance"
+	ScannerSubsystem    Subsystem = "scannerv4"
```

Comment:
Using "scannerv4" to not step on scanner v2 metrics if they exist (I haven't checked).